### DISCUSSION TITLE
While using SQL Server Express 13.0.4001, Encountered the following problems.

### DISCUSSION THREAD

**Discussion by Line4GitHub** [OrchardCMS/OrchardCore#10818] [General]
SqlException: Incorrect syntax near 'OFFSET'. Invalid usage of the option NEXT in the FETCH statement.
in next line of statment
IEnumerable<ContentItem> model = await query.ListAsync();

-----------------

Microsoft.Data.SqlClient.SqlCommand+<>c.<ExecuteDbDataReaderAsync>b__203_0(Task<SqlDataReader> result)
System.Threading.Tasks.ContinuationResultTaskFromResultTask<TAntecedentResult, TResult>.InnerInvoke()
System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, object state)
System.Threading.Tasks.Task.ExecuteWithThreadLocal(ref Task currentTaskSlot, Thread threadPoolThread)
Dapper.SqlMapper.QueryAsync<T>(IDbConnection cnn, Type effectiveType, CommandDefinition command) in SqlMapper.Async.cs
YesSql.Store.ProduceAsync<T, TState>(WorkerQueryKey key, Func<TState, Task<T>> work, TState state)
YesSql.Services.DefaultQuery+Query<T>.ListImpl()
YesSql.Services.DefaultQuery+Query<T>.ListImpl()
OrchardCore.CRUD.Controllers.HomeController.Index(PagerParameters pagerParameters) in HomeController.cs
-
				pager.PageSize = maxPagedCount;
			var routeData = new RouteData();
			var pagerShape = (await New.Pager(pager)).TotalItemCount(maxPagedCount > 0 ? maxPagedCount : await query.CountAsync()).RouteData(routeData);
			var pageOfContentItems = await query.Skip(pager.GetStartIndex()).Take(pager.PageSize).ListAsync();
			IEnumerable<ContentItem> model = await query.ListAsync();
			// Prepare the content items Summary Admin shape
			var contentItemSummaries = new List<dynamic>();
			foreach (var contentItem in pageOfContentItems)
			{
				contentItemSummaries.Add(await _contentItemDisplayManager.BuildDisplayAsync(contentItem, _updateModelAccessor.ModelUpdater, "Summary"));
Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor+TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, object controller, object[] arguments)
System.Threading.Tasks.ValueTask<TResult>.get_Result()
System.Runtime.CompilerServices.ValueTaskAwaiter<TResult>.GetResult()
Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask<IActionResult> actionResultValueTask)

**Comment by hishamco** [comment]
Please move your issue into [YesSQL](https://github.com/sebastienros/yessql) repo

**Reply by Line4GitHub** [reply]
Thank you very much for your explanation.

**Comment by deanmarcussen** [comment]
You want to check that you're actually attached to the SQL Express v13 server.

That error is one that occurs on SQL 2008, which is not currently supported. But v13 is 2016, so should support the paging / offsets we use

**Reply by Line4GitHub** [reply]
Thank you very much for your explanation!
